soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -536,10 +494,6 @@ void CheckTrackerTransition(uint32_t sceneNum) {
     if (!GameInteractor::IsSaveLoaded()) {
         return;
     }
-    gSaveContext;
-    if (transitionCheck) {
-        transitionCheck = false;
-    }
     doAreaScroll = true;
     previousArea = currentArea;
     currentArea = GetCheckArea();
```

Comment:
This is probably where we would check for SCENE_POTION_SHOP_KAKARIKO && IS_LINK_ADULT yea?

Reply:
In reference to what? Usually, if I need to differentiate, I differentiate where the functionality differs. But thanks for reminding me that I need to prevent the shop stuff from updating adult potion shop when entering as child XD.

Reply:
Yea that's what this was in reference to, more so the code below my comment this was just the lowest I could place the comment

Reply:
To further answer the question, though, the RCArea will still be Kak whether you're child or adult in the Kak potion shop, so the check actually went in SetShopSeen.